dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -99,7 +99,10 @@ export const decideAdSlot = (
 		collectionCount > minContainers &&
 		index === getMerchHighPosition(collectionCount)
 	) {
-		if (isInNetworkFrontsBannerTest || isInSectionFrontsBannerTest) {
+		if (
+			(isInNetworkFrontsBannerTest || isInSectionFrontsBannerTest) &&
+			!hasPageSkin
```

Comment:
I think this will hit the `else` condition and display a `merchandising-high` slot when there is a pageskin. Nesting the `hasPageSkin` check to inside this if condition and returning null if so, should work

Reply:
Thanks, actually I noticed we don't need any changes here at all 🤦‍♂️ This logic is hard to follow defo needs a refactor after the test!

Reply:
Yeah, agreed that it's confusing. It's tricky because it's temporary code that should exist for the duration of the two tests. Now, with the likely delay of the go-live date, this code might exist for a bit longer than initially anticipated.